pkgs/applications/networking/zerobin/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, python3Packages
+, pythonOlder
+, buildPythonApplication
+, fetchFromGitHub
+, nodePackages
+, doit
+}:
+
+buildPythonApplication rec {
+  pname = "zerobin";
+  version = "1.0.5";
+  src = fetchFromGitHub {
+    owner = "Tygs";
+    repo = "0bin";
+    rev = "v${version}";
+    sha256 = "1dfy3h823ylz4w2vv3mrmnmiyvf6rvyvsp4j3llr074w9id0zy16";
+  };
+
+  disabled = pythonOlder "3.7";
+
+  nativeBuildInputs = with python3Packages; [ doit pyscss nodePackages.uglify-js ];
+  propagatedBuildInputs = with python3Packages; [ appdirs beaker bleach bottle clize lockfile paste ];
+  prePatch = ''
+    # replace /bin/bash in compress.sh
+    patchShebangs .
+  '';
+  buildPhase = ''
+    doit build
+  '';
+
+  # zerobin has no check, but checking would fail with:
+  # nix_run_setup runserver: Received extra arguments: test
+  # See https://github.com/NixOS/nixpkgs/pull/98734#discussion_r495823510
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "A client side encrypted pastebin";
+    homepage = "https://0bin.net/";
+    license = licenses.wtfpl;
+  };
```

Comment:
🤔 My understanding is that all Nix packages are expected to have a maintainer, but the zerobin package was already without one. I'm not sure how the policy is applied in everyday practice, so I'll leave it to other reviewers.

Reply:
At this point I haven't used `zerobin` yet, so I'm not sure if I will care enough about it to add myself as a maintainer. For me this PR was just to give a helping hand to a friend testing `zerobin`.

AFAIK this is what it means to be a maintainer:
```
a maintainer will do a best effort to help resolving any issues
with the package explicitly brought to their attention,
and believes they can be useful in that. 
```
Source: https://logs.nix.samueldr.com/nixos-dev/2020-05-10#1589147927-1589147984;
But I have not searched the RFCs to see if there is something more.